packages/react/src/primitives/shared/responsive/__tests__/useBreakpoint.test.ts

```diff
@@ -69,7 +48,53 @@ describe('useBreakpoint', () => {
     expect(breakpoint).toBe('base');
   });
 
-  // Test going down and up the breakpoints
-  mediaQueries.forEach(testBreakpoints);
-  mediaQueries.reverse().forEach(testBreakpoints);
+  it.each(mediaQueries)(
+    `should return the $breakpoint breakpoint as expected`,
+    ({ query, breakpoint }) => {
```

Comment:
Couldn't this be the shared function here?

Reply:
Tried that but it gives a linting error in the `it` blocks because eslint thinks there are no tests inside. We could just disable that lint rule but thought it was probably okay to be more verbose since it's a test file